src/ThankYou.js

```diff
@@ -2,11 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const ThankYou = ( {switchView} ) => 
-				<div className="thankyou_main">
-					<h2>Thanks for participating. Your response has been submitted.</h2>
-					<div><button type="button" className="btn" onClick={switchView}>Return to Form</button></div>
-				</div>
-
+					<div className="container">
+					   <div className="thankyou_header">
+					   		<img src={require('./images/logo.png')} alt="logo" />
+				   		</div>
+				   		<div className="thankyou_main">
+							<img src={require('./images/ultimate-bridal-show.jpg')} alt="Ultimate Bridal Show" className="event_img" />
+							<p className="thankyou_message">Your entry has been submitted</p>
+							<div><button type="button" className="btn thankyou_button" onClick={switchView}>Submit Another Entry</button></div>
+						</div>
+					</div>
+				
 ThankYou.propTypes = {
```

Comment:
Naming components is a hard feat, so i get it, lets try to make the name describe what the component is doing as opposed to describing how its doing it.. so instead of naming this `ThankYou` I would name the component `Confirmation` or `ConfirmationView`